packages/block-library/src/social-links/index.js

```diff
@@ -25,16 +25,25 @@ export const settings = {
 	example: {
 		innerBlocks: [
 			{
-				name: 'core/social-link-wordpress',
-				attributes: { url: 'https://wordpress.org' },
+				name: 'core/social-link',
+				attributes: {
+					type: 'wordpress',
```

Comment:
Is this supposed to be "type" or "site"?

Reply:
Good catch.